src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// This is an extension method to be used with the <see cref="DnnImageFeaturizerEstimator"/> in order to use a pretrained ResNet18 model.
+    /// The NuGet containing this extension is also guaranteed to include the binary model file. Note that when building the project
+    /// containing this extension method, the corresponding binary model will be downloaded from the CDN at
+    /// https://aka.ms/mlnet-resources/image/ResNetPrepOnnx/ResNetPreprocess.onnx and
+    /// https://aka.ms/mlnet-resources/image/ResNet18Onnx/ResNet18.onnx  and placed into the local app directory
+    /// folder under mlnet-resources.
+    /// </summary>
+    public static class ResNet18Extension
+    {
+        /// <summary>
+        /// If including this through a NuGet, the location of the model will be the same as of this file. This looks for the model there.
```

Comment:
The XML summary comments on these methods should state what the method is doing. This type of comment is more appropriate for a `<remarks>` comment.